python/ray/serve/_private/vendored/get_asgi_route_name.py

```diff
@@ -0,0 +1,91 @@
+# This code is adapted from:
+# https://github.com/elastic/apm-agent-python/blob/main/elasticapm/contrib/starlette/__init__.py
```

Comment:
Nit: We might want to pin this to a commit 🙃

Reply:
do you mean the link?

Reply:
Yea, right now it's pointed to main and that can change over time

Reply:
so something like this https://github.com/elastic/apm-agent-python/blob/f570e8c2b68a8714628acac815aebcc3518b44c7/elasticapm/contrib/starlette/__init__.py

Reply:
good point done